app/Console/Commands/PromptMine/ExploreFeedCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Console\Commands\PromptMine;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use Baka\Enums\StateEnums;
+use Kanvas\Users\Models\Users;
+use Kanvas\Apps\Models\Apps;
+use Kanvas\Notifications\Templates\PromptMine\ExploreFeed;
+
+class ExploreFeedCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'promptmine:explore-feed {app_id}';
```

Comment:
@rwhite27 rename thi to email all user of a app, by passing the app id and the template they wan to use, follow using this same email logic

``` 
 $notification = new Blank(
                    $template,
                    ['membershipUpgradeUrl' => $url, 'app' => $this->app],
                    ['mail'],
                    $people
                );
                $notification->setSubject($subject);
             
                    Notification::route('mail', $mail)->notify($notification);
                    ```

Reply:
ok

Reply:
done